README.md

```diff
@@ -24,28 +23,12 @@ Package manager         : `pnpm`
 └── README.md
 ```
 
-## Packages
-
-### contracts
-
-The contracts needed for running a round can be found within the [contracts package](packages/contracts)
-
-##### Deploy Steps
-
-To know how the contracts should be setup, refer [DEPLOY_STEPS.md](packages/contracts/docs/DEPLOY_STEPS.md)
-
 
-##### Chain Deployment List
+## contracts
 
-To know the addresses are deployed on which network. refer [CHAINS.md](packages/contracts/docs/CHAINS.md)
+The contracts needed for running a round can be found within the [contracts package](https://github.com/Allo-Protocol/contracts)
```

Comment:
Nit: should we add something like "in the Allo-Protocol org"?

The contracts needed for running a round can be found within the [contracts package](https://github.com/Allo-Protocol/contracts) in the [Allo Protocol](https://github.com/Allo-Protocol) organisation